test/common/src/main/kotlin/org/codice/compliance/verification/binding/RedirectVerifier.kt

```diff
@@ -66,7 +65,12 @@ class RedirectVerifier(responseDom: Node, parsedResponse: Map<String, String>, g
      * Verifies the Signature and SigAlg according to the redirect binding rules in the binding spec
      * 3.4.4.1 DEFLATE Encoding
      */
-    fun verifyRedirectSignature(signature: String, samlResponse: String?, relayState: String?, sigAlg: String?) {
+    fun verifyRedirectSignature(response: IdpRedirectResponse) {
+        val signature = response.signature
```

Comment:
✏️ Options:
1. Just use the `response.X` values directly in the call below to `SimpleSign().validateSignature`
2. Wrap the call to `SimpleSign().validateSignature` with a `with(response)` block.

I think the latter might actually read a bit clumsy (but not too badly); the former is probably the most clear.

As opposed to the `verifyRedirectRelayState` method, where the local `val`s are used more than once, there really isn't any gain here by saving them off in a new handle.

Reply:
Good catch.

I chose to go with the first option.
The second option (wrapping with a `when(response)` block nested the `try` loop and looked too clunky. Plus the amount of space it saved with using `this.X` instead of `response.X` wasn't worth it.